src/table/components/TableWrapper.jsx

```diff
@@ -130,7 +130,7 @@ export default function TableWrapper(props) {
     borderStyle: 'solid',
     borderColor: theme.table.borderColor,
     height: '100%',
-    backgroundColor: theme.table.backgroundColor,
+    backgroundColor: theme.table.tableBackgroundColor ? theme.table.backgroundColor : 'inherit',
```

Comment:
if `tableBackgroundColor` exists, use `backgroundColor`?

Reply:
Although it does not affect the outcome, I will fix it.

Reply:
Fixed